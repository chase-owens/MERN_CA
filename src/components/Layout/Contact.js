import React from 'react';

import { theme } from 'styles/theme';

import { isMobile } from 'react-device-detect';
import Nav from '../Nav/Nav';
import ContactForm from '../ContactForm/ContactForm';
import ContactNav from '../ContactNav/ContactNav';
import Typography from '@material-ui/core/Typography';

const Contact = () => {
  return (
    <div style={{ background: theme.palette.primary.main, minHeight: '100vh' }}>
      <ContactNav />
      <Typography
        style={{
          fontSize: isMobile ? '2.5em' : '2em',
          padding: 20
        }}
        align='center'
        paragraph
        variant='display4'
      >
        Contact Us
      </Typography>
      <ContactForm />
    </div>
  );
};

export default Contact;
