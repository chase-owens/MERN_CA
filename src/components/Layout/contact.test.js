import React from 'react';
import Contact from './Contact';
import { shallow } from 'enzyme';

describe('Contact', () => {
  it('renders without crashing', () => {
    const contact = shallow(<Contact />);
    expect(contact).toMatchSnapshot();
  });
});
