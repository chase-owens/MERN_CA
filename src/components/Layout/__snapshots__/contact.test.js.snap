// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders without crashing 1`] = `
<div
  style={
    Object {
      "background": "#29bbf6",
      "minHeight": "100vh",
    }
  }
>
  <ContactNav />
  <WithStyles(Typography)
    align="center"
    paragraph={true}
    style={
      Object {
        "fontSize": "2em",
        "padding": "20px 0",
      }
    }
    variant="display4"
  >
    Contact Us
  </WithStyles(Typography)>
  <WithStyles(WithTheme(WithWidth(Connect(ContactForm)))) />
</div>
`;
